pyccel/codegen/python_wrapper.py

```diff
@@ -198,9 +216,18 @@ def create_shared_library(codegen,
                                                     output_folder = pyccel_dirpath,
                                                     sharedlib_modname = sharedlib_modname,
                                                     verbose = verbose)
+    compile_wrapper_time = time.time() - start_compile_wrapper
 
     # Change working directory back to starting point
     os.chdir(base_dirpath)
 
     # Return absolute path of shared library
-    return sharedlib_filepath
+    timings = {'Dependency compilation' : compile_libs_time,
+               'Wrapper creation' : wrapper_creation_time,
+               'Wrapper printing': print_cwrapper_time,
+               'Wrapper compilation': compile_wrapper_time}
+    if language == 'fortran':
+        timings['Bind C wrapping'] = bind_c_wrapping_time
+        timings['Bind C printing'] = bind_c_printing_time
+        timings['Bind C compiling'] = bind_c_compiling_time
```

Comment:
The new code should be added above the comment line `# Return absolute path of shared library`.

By the way, `timings` should be added to the `Returns` section of the function docstring.

Reply:
Fixed f679fca1

Reply:
I'm surprised it didn't complain